src/CTA.WebForms2Blazor/ProjectManagement/ProjectAnalyzer.cs

```diff
@@ -1,23 +1,23 @@
-﻿using System;
-using System.IO;
+﻿using System.IO;
 using System.Collections.Generic;
 
-namespace CTA.WebForms2Blazor
+namespace CTA.WebForms2Blazor.ProjectManagement
 {
-    public class WebFormsProjectAnalyzer
+    public class ProjectAnalyzer
     {
         private readonly string _inputProjectPath;
 
         public string InputProjectPath { get { return _inputProjectPath; } }
 
-        public WebFormsProjectAnalyzer(string inputProjectPath)
+        public ProjectAnalyzer(string inputProjectPath)
         {
             _inputProjectPath = inputProjectPath;
         }
 
         public IEnumerable<FileInfo> GetProjectFileInfo()
         {
-            throw new NotImplementedException();
+            var directoryInfo = new DirectoryInfo(_inputProjectPath);
+            return directoryInfo.GetFiles("*", SearchOption.AllDirectories);
         }
```

Comment:
Have you considered the case where we encounter build artifacts or other files we want to skip over? Not sure if you want to handle that here, elsewhere, or at all.

Reply:
Yeah we should definitely handle that, probably here as well, but not sure how we want to go about it yet. For now I'm just probably just going to put a todo comment in the method so we can come back to it

Reply:
Cool, sounds good. We can handle this after discussing potential options